index.html

```diff
@@ -8,22 +8,146 @@
   <link rel="stylesheet" href="style.css">
 </head>
 <body>
-  <header class="header-section">
-    <div class="bg-image"></div>
-    <nav>
-        <img src="images/menu.svg" alt="">
-    </nav>
-   </header>
-   <section id="second-main">
-     <h1>AMBROSE <br> ISIGBA</h1>
-     <p>Hello! I am a passionate software developer! my goal
-        is to help you build a quality product(s), features 
-        and/or website with international standard. You are 
-        highly invited to take a tour of my works. If you fall in 
-        love with  my work gallery, and you are planning for one or 
-        you need a code mentor. AMBROSE's CODELAB get you covered</p>
-      <button type="button">Start colaboration</button>
-      <div id="second-section-bg-img"></div>
+  <header id="header-section">
+      <div id="left-float"></div>
+      <nav id="right-float">
+        <a href="#">
+          <img src="images/logo-menu.svg" alt="">
+        </a>
+      </nav>
+  </header>  
+  <main id="section-main">
+        <h1>AMBROSE <br> ISIGBA</h1>
+        <p>Hello! I am a passionate software developer! my goal
+           is to help you build a quality product(s), features 
+           and/or website with international standard. You are 
+           highly invited to take a tour of my works.If you fall in 
+            love with  my work gallery, and you are planning for one or 
+           you need a code mentor. AMBROSE's CODELAB get you covered</p>
+        <button type="button">Start colaboration</button>
+        <div id="second-section-bg-img"></div>
+  </main>
+    <!-- project section -->
+  <section>
+    <div id="project-container">
```

Comment:
- [ ] your semantic structure is pretty great. What I recommend here is to just remove the `div `with project-container `id`. just pass that id to the `section `itself, it will not mess up your project.  I suggested it because your display grid is supposed to to be on your section container, not the child `div`.